src/src/com/microsoft/aad/adal/AuthenticationContext.java

```diff
@@ -1672,15 +1664,9 @@ private boolean validateAuthority(final URL authorityUrl) {
 
             // Set CorrelationId for Instance Discovery
             mDiscovery.setCorrelationId(getRequestCorrelationId());
-            try {
-                boolean result = mDiscovery.isValidAuthority(authorityUrl);
-                Logger.v(TAG, "Finish validating authority:" + authorityUrl + " result:" + result);
-                return result;
-            } catch (Exception exc) {
-                Logger.e(TAG, "Instance validation returned error", "",
-                        ADALError.DEVELOPER_AUTHORITY_CAN_NOT_BE_VALIDED, exc);
-
-            }
+            boolean result = mDiscovery.isValidAuthority(authorityUrl);
```

Comment:
What that means is the app should be able to catch and recover if it so chooses.  That implies that the exception that ADAL lets through should indicate in a meaningful way the cause of the problem.  If that is what is going on in this change then I am fine with it.